src/time_series_metadata_store.jl

```diff
@@ -275,6 +277,50 @@ function _migrate_from_v2_4(store::TimeSeriesMetadataStore)
     _add_migrated_rows!(store, new_rows)
 end
 
+"""
+The Python package infrasys stores resolutions in the most human-readable format, such as
+P0DT1H for 1 hour and P0DT5M for 5 minutes.
+InfrastructureSystems.jl always stores regular resolutions in milliseconds.
+InfrastructureSystems.jl does this because user queries might include resolution and we don't
+want to have to search for any conceivable resolution.
+This will go away if we change infrasys to match.
+"""
+function _handle_infrasys_resolutions(store::TimeSeriesMetadataStore)
+    hours_regex = r"^P0DT\d+H$"
+    minutes_regex = r"^P0DT\d+M$"
+    seconds_regex = r"^P0DT\d+S$"
```

Comment:
This doesn't handle days or weeks. If this solution is temporary, no big deal. If it is to be permanent, we should expand this logic or implement a different solution.